experimental/GroebnerWalk/test/runtests.jl

```diff
@@ -0,0 +1 @@
+include("standard_walk.jl")
```

Comment:
this file is redundant, as Oscar automatically runs all files from the test directory when running tests

Reply:
runtests.jl is deleted